core/basemol.py

```diff
@@ -398,14 +398,17 @@ class Molecule (object):
     """
 
 
-    def __init__(self, filename=None, inputformat=None, geometry=1, **other):
+    def __init__(self, filename=None, inputformat=None, geometry=1, read_options=None, **other):
```

Comment:
Initializing `self.settings` with `**other` seems like a dead and useless feature to me. I thought that maybe it would be nicer to use constructor's `**other` for gathering `read_options`? In that case `geometry` should be one of them. That would make the whole setup a bit cleaner. Whaddya think?

Reply:
I think the same, just thought you put it there for a reason. I just dorpped all the `geometry` options and moved them to `other`. I updated the doc too.

Also, since this was bugging me all the time: I defaulted the `frame` option in the readers to zero. Starting a selection with 1 instead of 0 is very unpythonic and confusing in my opinion....